pom.xml

```diff
@@ -1030,6 +1036,10 @@
             <groupId>javax.ws.rs</groupId>
             <artifactId>jsr311-api</artifactId>
           </exclusion>
+          <exclusion>
+            <groupId>org.eclipse.jetty</groupId>
+            <artifactId>jetty-webapp</artifactId>
+          </exclusion>
```

Comment:
The `hadoop-3.2`-specific exclusion starts from https://github.com/apache/spark/pull/26742/files#diff-600376dffeb79835ede4a0b285078036R1034 .